imports/api/groceries.js

```diff
@@ -38,6 +40,14 @@ Meteor.methods({
 
 		return Groceries.remove(groceryId);
 	},
+	'groceries.archive'(groceryId, archived = true) {
```

Comment:
I'm adding the ability to "de-archive" a grocery item here, though I don't think we need to make that part of the MVP. But if we do want to implement "de-archiving" in the future, then we can